scripts/uart_terminal/scripts/get_serial_ports.py

```diff
@@ -20,9 +23,12 @@ def get_serial_ports():
 
     ports = list()
 
-    for line in output_decoded_lines:
-        if "VCOM0" in line:
-            info = line.split("    ")
-            ports.append(info[1])
+    if len(output_decoded_lines) > 0:
+        print("%6s %11s" % ("ID", "Port"))
```

Comment:
Could you elaborate on this formatting?

Reply:
Not sure what you are asking for here, the ID is the board id and the port is the com port it is attached to for VCOM0.
Can you elaborate on what is requested here?

Reply:
Probably will have a quick look

Reply:
Will set the positions with regard to the ID and the port name lengths.